### ISSUE TITLE
Inquiry Regarding Reproduction of Results for OCTA-6M Dataset

### ISSUE THREAD

**Issue by paantea** [coreeey/OCT2Former#4]
Hi,

I hope this message finds you well. I am currently working with your repository, OCT2Former , available at https://github.com/coreeey/OCT2Former , and I would like to express my gratitude for making the code publicly available. It has been incredibly helpful in understanding and reproducing your work.

I have successfully reproduced the results reported in your paper for the OCTA-3M dataset, achieving performance metrics that align exactly with those in your study. However, I encountered challenges when attempting to reproduce the results for the OCTA-6M dataset. Specifically, the performance metrics obtained from running the provided script for OCTA-6M are significantly lower than those reported in the paper.

Here is the command I used to run the training script for OCTA-6M :

"python train.py \
--dataset='OCTA-6M' \
--num_epochs=100 \
--dataset_file_list='utils/OCTA_6M.csv' \
--data_root='OCTA_6M/Projection Maps/OCTA(ILM_OPL)' \
--data_root_aux='OCTA_6M/Projection Maps/OCT(ILM_OPL)' \
--target_root='OCTA_6M/GroundTruth' \
--run_dir='6M' \
--in_channel=2 \
--batch_size=2 \
--lr=5e-4 \
--img_aug \
--cuda_id=6"
While the setup appears consistent with the instructions in the README, the results do not match the performance described in the paper. To better understand the discrepancy, I would greatly appreciate clarification on the following points:

Dataset-Specific Preprocessing : Are there any preprocessing steps specific to the OCTA-6M dataset (e.g., resizing from 400*400 to 304*304)that differ from those used for OCTA-3M ?
Hyperparameter Tuning : Were any hyperparameters (e.g., learning rate, batch size, auxiliary loss weight, class weights) adjusted specifically for OCTA-6M ?
Model Architecture : Were there any modifications to the model architecture (e.g., patch size, token dimensions, network depths) for OCTA-6M compared to OCTA-3M ?
Additional Settings : Are there any other settings or considerations specific to OCTA-6M that are not mentioned in the README but are necessary to reproduce the reported results?
If there are any additional details or unpublished modifications to the codebase that were used for OCTA-6M , I would greatly appreciate your guidance. Ensuring reproducibility is crucial for validating and building upon your work, and I am eager to resolve this issue.

Thank you for your time and assistance. I look forward to your response.

Best regards,

**Comment by coreeey** [comment]
Sorry for the inconvenience, Paantea. This project was done during my academic years two years ago, and unfortunately, I can’t seem to locate the relevant records anymore. After thinking about it, the issue might be related to the seed or the number of epochs. You can try replacing the seed with another value or modifying the number of epochs to 50 and see if there are any changes. The number of epochs mainly affects the learning rate adjustments.

Additionally, since this project is relatively old, while it may be useful for learning purposes, I wouldn’t recommend delving too deeply into it for advanced research. If you’re looking for a better baseline to work with, I recommend using the following repository:
https://github.com/ShellRedia/SAM-OCTA

Hope this helps!
